### DISCUSSION TITLE
Ayuda con el push desde la terminal

### DISCUSSION THREAD

**Discussion by ascamarena** [LaunchX-InnovaccionVirtual/CursoIntroPython#129] [Q&A] [answered]
Después de `git remote add origin <URL>`  y `git push origin master` me aparece ese error. Supongo que lo primero se arregla pasando por mi token pero el segundo mensaje aparece <b>fatal</b>.

![image](https://user-images.githubusercontent.com/92229619/153620352-0a3c0615-79e2-42de-9ef0-f869f4e61e2a.png)

¿Ese error de autenticación se debe a que no tengo el token o a que no encontró el repo?  Ayuda 😢

Anexo imágenes del código previo:
![image](https://user-images.githubusercontent.com/92229619/153621666-9c4e5991-1c0e-461f-9167-da7e91f825ac.png)

Si llegaste hasta aquí, ten un buen día ☀

<p align="center">
<img src="https://media.giphy.com/media/AviNPoJQugg4ege4A1/giphy.gif">
</p>

**Comment by doguedogue** [comment, marked as answer]
Hola, es fácil, lo que te indica es que no te va a dejar subir por usuario/password tienes que usar SSH.

Aquí los pasos: [https://docs.github.com/es/authentication/connecting-to-github-with-ssh](https://docs.github.com/es/authentication/connecting-to-github-with-ssh)

Para hacerlo más rápido podrías subir directamente desde github en add files -->upload files a tu repositorio.
![image](https://user-images.githubusercontent.com/23409026/153626952-47022db7-aa8b-41ec-a918-fc9b18f71b0c.png)


Espero que te sirva!

**Reply by Cesargj** [reply]
gracias, tenia el mismo problema
